setup.py

```diff
@@ -69,8 +69,6 @@ def get_advanced_templates():
             'pkgpanda=pkgpanda.cli:main',
             'mkpanda=pkgpanda.build.cli:main',
             'dcos_installer=dcos_installer.cli:main',
```

Comment:
Nothing new. There were trailing commas before and it seemed to work alright.
Happy to remove a comma though, if requested.